.github/workflows/ci-additional.yml

```diff
@@ -75,7 +73,7 @@ jobs:
 
       - name: Run tests
         shell: bash -l {0}
-        run: source continuous_integration/scripts/run_tests.sh
+        run: pytest -v --doctest-modules --ignore-glob='*/test_*.py' dask
```

Comment:
I'm not totally sure why, but running the command directly here instead of through `continuous_integration/scripts/run_tests.sh` ensures that we ignore test files

Reply:
huh! Well nice detective work :)